mozilla-mobile/android-components/components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -170,6 +170,8 @@
     <string name="mozac_feature_addons_successfully_installed">Successfully installed %1$s</string>
     <!-- Text shown after failed to install an add-on. %1$s is the add-on name. -->
     <string name="mozac_feature_addons_failed_to_install">Failed to install %1$s</string>
+    <!-- Text shown when attempting to install a blocklisted add-on. -->
```

Comment:
@willdurand: Please always add a comment in any string that introduces a variable/placeholder, describing what the variable will be replaced with.
More info on l10n dev best practices [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#add-localization-notes). Thanks

Reply:
ha yes, sorry about that.

Reply:
I submitted a fix: https://github.com/mozilla-mobile/firefox-android/pull/3115

Reply:
Perfect, thanks a bunch!